concent_api/core/views.py

```diff
@@ -310,6 +315,30 @@ def receive_out_of_band(request, _message):
             )
             message_verdict.sig = None
             return message_verdict
+        if last_undelivered_receive_status.type == 'RejectReportComputedTask':
+            serialized_reject_report_computed_task = last_undelivered_receive_status.data.tobytes()
+            try:
+                decoded_reject_report_computed_task = load(
```

Comment:
Shouldn't this be `deserialize()` now?

Reply:
All changes about deserialization are in fix-incorrect-key-changes branch.

Reply:
`fix-incorrect-key-changes` is now merged. So this needs to be changed now.